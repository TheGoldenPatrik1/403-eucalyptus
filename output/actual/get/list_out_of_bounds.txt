Error on line 2: Index out of bounds: 1 for List x
