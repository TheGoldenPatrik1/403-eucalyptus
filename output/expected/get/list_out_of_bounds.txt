Error on line 2: Index out of bounds: 1 for list x